src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -146,6 +168,8 @@ class AndroidDeviceControllerWrapper : public chip::Controller::DevicePairingDel
     jbyteArray operationalDatasetBytes = nullptr;
     jbyte * operationalDataset         = nullptr;
 
+    chip::Controller::AutoCommissioner mAutoCommissioner;
```

Comment:
We wouldn't need to have a separate AutoCommissioner instance here if we had the "ask for credentials" step, right?

Reply:
Currently, that is correct